proc.go

```diff
@@ -378,3 +383,21 @@ func (p *Proc) Screenshot(fname string) error {
 
 	return nil
 }
+
+// RandomSeed changes the sequence of numbers generated by Random.
+func (p *Proc) RandomSeed(seed uint64) {
+	p.rand.Seed(seed)
+}
+
+// Random returns a pseudo-random number in [min,max).
+// Random will produce the same sequence of numbers every time the program runs.
+// Use RandomSeed with a seed that changes (like time.Now().UnixNano()) in order to
+// produce different sequence of numbers.
+func (p *Proc) Random(min, max float64) float64 {
+	return p.rand.Float64()*(max-min) + min
+}
+
+// RandomGaussian returns a random number fitting a Gaussian (normal) distribution.
```

Comment:
```suggestion
// RandomGaussian returns a random number following a Gaussian distribution with the provided
// mean and standard deviation.
```

(stricly speaking, the normal distribution is with mean=0 and stddev=1)